python/lsst/cp/verify/verifyStats.py

```diff
@@ -472,6 +472,7 @@ def crImageStats(self, exposure, statControl):
             crMask = crRejectedExp.getMaskedImage().getMask().getPlaneBitMask("CR")
             spans = afwGeom.SpanSet.fromMask(crRejectedExp.mask, crMask)
             spans = spans.dilated(self.config.crGrow)
+            spans = spans.clippedTo(crRejectedExp.getBBox())
```

Comment:
It clips the grown set of spans to the bounding box of the image.  The error was arising because a CR located at (0, 0) grown isotropically by one pixel would include {(-1, -1), (-1, 0), (0, -1)}.  Those all lie outside the image, so applying that as a mask was triggering the error.  I think this is formally restricting the spanset to the intersection of the initial set (L474) and the image bbox.